tools/downloader/list_topologies.yml

```diff
@@ -1,3480 +0,0 @@
-# Copyright (c) 2018 Intel Corporation
-
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-
-#      http://www.apache.org/licenses/LICENSE-2.0
-
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-topologies:
-  - name: "densenet-121"
```

Comment:
this is always replaced by unique folder name, right?

Reply:
Yes.